diff --git a/menu_views.admin.inc b/menu_views.admin.inc
index f5b05ec..f3a2cb7 100644
--- a/menu_views.admin.inc
+++ b/menu_views.admin.inc
@@ -139,6 +139,7 @@ function _menu_views_form_tokens_ui(array &$element) {
  *   The complete form state array passed by reference.
  */
 function _menu_views_form_alter(array &$element, array &$form, array &$form_state) {
+  unset($form['#token']);
   // Still need to render the form, just hide it for those who don't have access.
   $access = user_access('administer menu views');
   // Only need to attach or alter form if user has access.
@@ -731,4 +732,4 @@ function _menu_views_parents_recurse($tree, $menu_name, $indent, &$options, $exc
       }
     }
   }
-}
\ No newline at end of file
+}
